client/ayon_houdini/plugins/publish/validate_wait_for_render.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+import hou
+
+import pyblish.api
+from ayon_core.pipeline import PublishValidationError
+from ayon_core.pipeline.publish import RepairAction
+
+from ayon_houdini.api import plugin
+
+
+class EnableWaitForRender(RepairAction):
+    label = "Enable WaitForRendertoComplete"
+    icon = "mdi.pencil-plus-outline"
```

Comment:
```suggestion

```

Reply:
implemented here, Thank you
https://github.com/ynput/ayon-houdini/pull/64/commits/9f2c93acaa9b78fe05674bef2517d2ae02b30f49